src/test/java/org/folio/rest/impl/PoNumberTest.java

```diff
@@ -29,6 +47,19 @@ public void testGetPoNumberOk() throws MalformedURLException {
     assertEquals(1, poNumber2 - poNumber1);
   }
 
+  @Test
+  public void testError() throws Exception {
+    AutoCloseable mockitoMocks = MockitoAnnotations.openMocks(this);
+    OrderPostgresDAO orderPostgresDAO = new OrderPostgresDAO();
+    doReturn(0).when(rowSet).rowCount();
+    doReturn(Future.succeededFuture(rowSet)).when(conn).execute(anyString());
+    Future<Long> f = orderPostgresDAO.getNextPoNumber(conn);
+    assertThat(f.failed(), is(true));
+    HttpException thrown = (HttpException)f.cause();
+    assertEquals(Response.Status.INTERNAL_SERVER_ERROR.getStatusCode(), thrown.getCode());
+    mockitoMocks.close();
+  }
```

Comment:
can we refactor the code by adding line. it will be easier to understand the flow

Reply:
Added blank lines.